src/main/locales/da/bf_locale.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "drivers/osd.h"
+
+#define LOCALE                        "da"                                  	// Max length:  2; Current language in short form: en, da, es, fr, nl ...;
+#define STR_LOCALE_SETUP              "Sprog:"                              	// Max length: 10; translation of: Language:;
+#define STR_COMMA                     ","                                   	// Max length:  1; translation of: Comma string ,;
+#define STR_PERIOD                    "."                                   	// Max length:  1; translation of: Period string .;
+#define STR_THOUSAND                  "."                                   	// Max length:  1; translation of: Thousand delimiter .;
```

Comment:
Well, I belive is was used in cli.c, but since nothing are translated there anymore, is can be deleted. 
Removed in bf_locale.json file, the source for bf_locale.h